exporters/ibmmq/exporter.yml

```diff
@@ -1,7 +1,7 @@
 # name of the exporter, should match with the folder name
 name: ibmmq
 # version of the package created
-version: 0.4.2
+version: 0.4.3
```

Comment:
I would add it as well in the template config the scrape_timeout commented out

Reply:
We can discuss [here](https://github.com/newrelic/newrelic-prometheus-exporters-packages/pull/165#discussion_r1121278759)

Reply:
Updated here: [1fe9b1a](https://github.com/newrelic/newrelic-prometheus-exporters-packages/pull/165/commits/1fe9b1aa46524eff8668b640368b541d8c584c0a)